pkg/sanity/sanity.go

```diff
@@ -63,6 +63,8 @@ var _ = BeforeSuite(func() {
 	By("creating mount and staging directories")
 	err = createMountTargetLocation(config.TargetPath)
 	Expect(err).NotTo(HaveOccurred())
+	err = createMountTargetLocation(config.StagingPath)
```

Comment:
What if staging path is not needed and it is not set. We should check here and avoid this section, right?

Reply:
resolved